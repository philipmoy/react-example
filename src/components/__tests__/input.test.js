import React from 'react';
import { shallow } from 'enzyme';

import CustomInput from '../input';

describe('CustomInput', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <CustomInput />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
