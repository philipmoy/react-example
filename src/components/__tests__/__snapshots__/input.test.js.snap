// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomInput renders correctly 1`] = `
<div
  className="custom-input"
>
  <input
    onChange={[Function]}
    onKeyDown={[Function]}
    value=""
  />
  <button
    onClick={[Function]}
    type="button"
  >
    <img
      alt="submit"
      src="/images/submit.jpg"
    />
  </button>
</div>
`;
